api/.dockerignore

```diff
@@ -1,3 +1,2 @@
 README
 Dockerfile
-node_modules
```

Comment:
This line is needed to keep the Docker image small - relevant for faster transfer times when deploying the stack. Is it necessary to remove it?

Reply:
This is definitely work in progress and if we move forward with it we should have different files for local and remote. Good that you spotted this.